src/luascript.cpp

```diff
@@ -9988,6 +9988,7 @@ int LuaScriptInterface::luaPlayerSetGhostMode(lua_State* L)
 
 	Tile* tile = player->getTile();
 	const Position& position = player->getPosition();
+	const bool isInvisible = player->isInvisible();
```

Comment:
whats the point of declaring this if its gonna be used only once anyway?

Reply:
So isInvisible() doesn't get called multiple times while iterating through the spectators, that would be wasteful

Reply:
true, didn't see the loop.